packages/block-editor/src/components/text-decoration-control/README.md

```diff
@@ -0,0 +1,39 @@
+# TextDecorationControl
+
+![TextDecorationControl Element in Inspector Control](https://raw.githubusercontent.com/WordPress/gutenberg/HEAD/docs/assets/text-decoration-component.png?raw=true)
+
+## Table of Contents
+
+1. [Usage](#usage)
+3. [Properties](#props)
```

Comment:
I'm not sure we need the ToC given its a small readme and other readmes don't appear to include it.